src/client.rs

```diff
@@ -154,12 +154,10 @@ fn deserialize_entity_mappings(
         let client_entity = deserialize_entity(cursor)?;
 
         if let Some(entry) = entity_map.to_client().get(&server_entity) {
-            if *entry == client_entity {
-                warn!("ignoring mapping from {server_entity:?} to {client_entity:?} that already exists");
-                continue;
-            } else {
+            if *entry != client_entity {
                 panic!("received mapping from {server_entity:?} to {client_entity:?}, but already mapped to {entry:?}");
             }
+            // while acks are in-flight you will receive the same mappings in multiple packets
```

Comment:
I would suggest to rewrite the comment like this. What do you think?
And could you put it on top of `if`?
```suggestion
            // It's possible to receive the same mappings in multiple packets if the server has not yet received ack from the client.
```